services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.test.tsx

```diff
@@ -1266,7 +1314,37 @@ describe('RateDetails', () => {
 
             await waitFor(() => {
                 expect(
-                    screen.getAllByText('You must upload at least one document')
+                    screen.getAllByText('You must upload a rate certification')
+                ).toHaveLength(2)
+
+                expect(continueButton).toHaveAttribute('aria-disabled', 'true')
+            })
+        })
+
+        it('disabled with alert if previously submitted with more than one rate cert file', async () => {
+            renderWithProviders(
+                <RateDetails
+                    draftSubmission={mockContractAndRatesDraftWithDocuments()}
+                    updateDraft={jest.fn()}
+                    previousDocuments={['testFile.docx', 'testFile.pdf']}
+                />,
+                {
+                    apolloProvider: {
+                        mocks: [fetchCurrentUserMock({ statusCode: 200 })],
+                    },
+                }
+            )
+
+            const continueButton = screen.getByRole('button', {
+                name: 'Continue',
+            })
+            expect(continueButton).not.toHaveAttribute('aria-disabled')
+
+            continueButton.click()
+
+            await waitFor(() => {
+                expect(
+                    screen.getAllByText('You must upload a rate certification')
```

Comment:
I think this should be:

```suggestion
                    screen.getAllByText('Only one document is allowed for a rate certification. You must remove documents before continuing.')
```